cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress", "node"]
```

Comment:
I needed to add `"node"` to the `"types"` array to get VSCode to stop complaining about not finding the types it needed.
The config probably could do with a review/edit. 😉

Reply:
Wouldn't be better to add this to the main `tsconfig.json`? Not really sure what's the best approach, I need to double-check it.

Reply:
For your information: I based this work on the [Cypress docs on working with TypeScript, configure tsconfig.json](https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfig-json) section.

I'll let you figure out what the best solution is. 😉